WordPress/src/main/java/org/wordpress/android/datasets/ReaderPostTable.java

```diff
@@ -270,11 +270,15 @@ public static void addOrUpdatePost(ReaderPost post) {
     }
 
     public static ReaderPost getPost(long blogId, long postId, boolean excludeTextColumn) {
+        return getPost(false, blogId, postId, excludeTextColumn);
+    }
 
+    public static ReaderPost getPost(boolean isFeed, long blogOrFeedId, long postOrItemId, boolean excludeTextColumn) {
```

Comment:
Aha, I had that like that but opted for the overloaded form. Not prob, I can split it up. Thanks!